scripts/contract_extensions/TestNetChainIds.sol

```diff
@@ -16,4 +16,5 @@ library TestNetChainIds {
   uint256 constant POLYGON_ZK_EVM_GOERLI = 1442;
   uint256 constant GNOSIS_CHIADO = 10200;
   uint256 constant SCROLL_SEPOLIA = 534351;
+  uint256 constant CELO_ALFAJORES = 44787;
```

Comment:
Should it be also the celo adapter deployment script here?

Reply:
for now i only added the network to have it, as i am basing all other ones on this pr. (This way on lz we can already add the path)